src/Build/Resources/xlf/Strings.de.xlf

```diff
@@ -2059,61 +2059,61 @@ Auslastung:          {0} Durchschnittliche Auslastung: {1:###.0}</target>
+        <target state="translated">MSB4244: Die SDK-Resolverassembly "{0}" konnte nicht geladen werden. {1}</target>
         <note>{StrBegin="MSB4244: "}</note>
       </trans-unit>
       <trans-unit id="CouldNotRunSdkResolver">
         <source>MSB4242: The SDK resolver "{0}" failed to run. {1}</source>
-        <target state="new">MSB4242: The SDK resolver "{0}" failed to run. {1}</target>
+        <target state="translated">MSB4242: Der SDK-Resolver "{0}" konnte nicht ausgeführt werden. {1}</target>
         <note>{StrBegin="MSB4242: "}</note>
       </trans-unit>
       <trans-unit id="CouldNotRunNuGetSdkResolver">
         <source>MSB4243: The NuGet-based SDK resolver failed to run because NuGet assemblies could not be located.  Check your installation of MSBuild or set the environment variable "{0}" to the folder that contains the required NuGet assemblies. {1}</source>
-        <target state="new">MSB4243: The NuGet-based SDK resolver failed to run because NuGet assemblies could not be located.  Check your installation of MSBuild or set the environment variable "{0}" to the folder that contains the required NuGet assemblies. {1}</target>
+        <target state="translated">MSB4243: Fehler beim NuGet-basierten SDK-Resolver, weil die NuGet-Assemblys nicht gefunden wurden. Überprüfen Sie Ihre Installation von MSBuild, oder legen Sie die Umgebungsvariable "{0}" auf den Ordner fest, der die erforderlichen NuGet-Assemblys enthält. {1}</target>
         <note>{StrBegin="MSB4243: "}</note>
       </trans-unit>
       <trans-unit id="ProjectImportSkippedInvalidFile">
         <source>Project "{0}" was not imported by "{1}" at ({2},{3}), due to the file being invalid.</source>
-        <target state="new">Project "{0}" was not imported by "{1}" at ({2},{3}), due to the file being invalid.</target>
-        <note></note>
+        <target state="translated">Das Projekt "{0}" wurde von "{1}" nicht um ({2},{3}) importiert, weil die Datei ungültig ist.</target>
+        <note />
       </trans-unit>
       <trans-unit id="ProjectImportSkippedMissingFile">
         <source>Project "{0}" was not imported by "{1}" at ({2},{3}), due to the file not existing.</source>
-        <target state="new">Project "{0}" was not imported by "{1}" at ({2},{3}), due to the file not existing.</target>
-        <note></note>
+        <target state="translated">Das Projekt "{0}" wurde von "{1}" nicht um ({2},{3}) importiert, weil die Datei nicht vorhanden ist.</target>
+        <note />
       </trans-unit>
       <trans-unit id="ErrorWritingProfilerReport">
         <source>MSBUILD : error MSB4239: Error writing profiling report. {0}</source>
-        <target state="new">MSBUILD : error MSB4239: Error writing profiling report. {0}</target>
-        <note></note>
+        <target state="translated">MSBUILD: Fehler MSB4239: Fehler beim Schreiben des Profilerstellungsberichts. {0}</target>
```

Comment:
should be `Profilerbericht` oder `Profilingbericht` instead of `Profilerstellungsberichts`.